examples/randompolymat/JuMP.jl

```diff
@@ -0,0 +1,65 @@
+#=
+find a polynomial matrix Q such that Q - H is SOS when H is random and the volume under the polynomials in Q is minimized
+=#
+
+struct RandomPolyMatJuMP{T <: Real} <: ExampleInstanceJuMP{T}
+    n::Int
+    halfdeg::Int
+    R::Int
+    use_wsos::Bool
+    use_matrixwsos::Bool
+    use_sdp::Bool
```

Comment:
this option isn't really used, can it be removed?

Reply:
yes, but I was being consistent with expdesign where ther eare three possible objectives but we still have a boolean for each